### DISCUSSION TITLE
[question] any plan for a "dart only" implementation for mobile?

### DISCUSSION THREAD

**Discussion by iapicca** [invertase/flutterfire_desktop#10] [Q&A] [answered]
As for the title
`is there any plan for a "dart only" implementation for mobile?`

**Comment by Ehesp** [comment, marked as answer]
If a package supports it then it's possible (such as Cloud Functions), but we won't be able to replicate the full native SDKs since services such as FCM, Crashlytics etc need some form of native integration.
